tgui-next/.eslintrc.yml

```diff
@@ -0,0 +1,118 @@
+parser: babel-eslint
+parserOptions:
+  ecmaVersion: 2019
+  sourceType: module
+  ecmaFeatures:
+    jsx: true
+env:
+  browser: true
+  node: true
+rules:
+  ## Possible Errors
+  # for-direction: error
+  # getter-return: error
+  no-async-promise-executor: error
+  # no-await-in-loop: error
+  # no-compare-neg-zero: error
+  no-cond-assign: error
+  # no-console: error
+  # no-constant-condition: error
+  # no-control-regex: error
+  # no-debugger: error
+  no-dupe-args: error
+  no-dupe-keys: error
+  no-duplicate-case: error
+  # no-empty: error
+  no-empty-character-class: error
+  no-ex-assign: error
+  no-extra-boolean-cast: error
+  # no-extra-parens: warn
+  no-extra-semi: error
+  no-func-assign: error
+  no-import-assign: error
+  no-inner-declarations: error
+  no-invalid-regexp: error
+  no-irregular-whitespace: error
+  no-misleading-character-class: error
+  no-obj-calls: error
+  no-prototype-builtins: error
+  no-regex-spaces: error
+  no-sparse-arrays: error
+  no-template-curly-in-string: error
+  no-unexpected-multiline: error
+  # no-unreachable: warn
+  no-unsafe-finally: error
+  no-unsafe-negation: error
+  # require-atomic-updates: error
+  use-isnan: error
+  valid-typeof: error
+
+  ## Best practices
+  complexity: [error, { max: 50 }] ## That is a VERY generous limit
+  curly: [error, all]
+  eqeqeq: [error, always]
+  dot-location: [error, property]
+  no-empty-pattern: error
+  no-multi-spaces: warn
+  no-octal: error
+  no-octal-escape: error
+  no-return-assign: error
+  no-self-assign: error
+  no-sequences: error
+  no-unused-labels: warn
+  no-useless-escape: warn
+  no-with: error
+  radix: error
+
+  ## Code style
+  array-bracket-newline: [error, consistent]
+  array-bracket-spacing: [error, never]
+  block-spacing: [error, always]
+  brace-style: [error, stroustrup, { allowSingleLine: false }]
```

Comment:
Please set to `1tbs`

Reply:
perhaps we can just simply relax some of these rules, including this one